models/location_helpers.go

```diff
@@ -231,7 +237,8 @@ type ChangeBackupLocationParams struct {
 
 // ChangeBackupLocation updates existing location by specified locationID and params.
 func ChangeBackupLocation(q *reform.Querier, locationID string, params ChangeBackupLocationParams) (*BackupLocation, error) {
-	if err := params.Validate(); err != nil {
+	configCount, err := params.Validate()
```

Comment:
not sure you need to know how many configs are set, is set or not would be enough.